api/controllers/communicator.js

```diff
@@ -145,7 +145,12 @@ function removeCommunicator(req, res) {
         error: err.message
       });
     }
-
+    if (!communicator) {
+      return res.status(404).json({
+        message: 'communicator not found. communicator Id: ' + id,
```

Comment:
Small detail: I would write "Communicator" instead of "communicator". Same thing for the `error` property.

Reply:
Thanks for your correction! I will solve it